client/scss/utilities/all.scss

```diff
@@ -0,0 +1,4 @@
+@import 'utilities/variables';
+@import 'utilities/mixins';
+@import 'utilities/functions';
+@import 'utilities/root-scope-classes';
```

Comment:
hmmmm… looks like I lied about having underscored _all_ our other scss. Most of it.